### DISCUSSION TITLE
unit:EINSTEIN

### DISCUSSION THREAD

**Discussion by dr-shorthair** [qudt/qudt-public-repo#1450] [General]
a Mole of Photons - see https://en.wikipedia.org/wiki/Einstein_(unit)

**Comment by steveraysteveray** [comment]
Another complicated unit. As the wikipedia article says, 

The einstein is a unit defined as the energy in one mole of photons (6.022×1023 photons).[1][2] Because energy is inversely proportional to wavelength, the unit is frequency dependent.

So, it's not possible to simply convert to Joules. Will need to wait on our support of a generalized expression for unit conversions.
